src/net/tmwa/serverfeatures.cpp

```diff
@@ -229,4 +229,9 @@ bool ServerFeatures::haveTeamId() const
     return tmwServerVersion > 0;
 }
 
+bool ServerFeatures::haveNewGuild() const
+{
+    return false;
```

Comment:
here should be
return tmwServerVersion > 0;

This mean it will be false for tmwa before olli changes, and true for any his version and newer